operator/src/test/java/oracle/kubernetes/weblogic/domain/DomainTestBase.java

```diff
@@ -400,6 +401,46 @@ public void whenBothClusterAndDomainSpecified_maxConcurrentStartupFromCluster()
         equalTo(1));
   }
 
+  @Test
+  public void afterMaxConcurrentShutdownSetForCluster_canReadIt() {
+    configureCluster("cluster1").withMaxConcurrentShutdown(3);
+
+    assertThat(domain.getMaxConcurrentShutdown("cluster1"), equalTo(3));
+  }
+
+  @Test
+  public void whenNotSpecified_maxConcurrentShutdownHasDefault() {
+    configureCluster("cluster1");
+    configureDomain(domain).withMaxConcurrentShutdown(null);
+
+    assertThat(domain.getMaxConcurrentShutdown("cluster1"),
+            equalTo(DEFAULT_MAX_CLUSTER_CONCURRENT_SHUTDOWN));
+  }
+
+  @Test
+  public void whenNotSpecified_maxConcurrentShutdownFromDomain() {
+    configureCluster("cluster1");
```

Comment:
This line should not be needed. The point is that the cluster is not configured.